presto-main/src/main/java/io/prestosql/sql/planner/planprinter/WindowOperatorStats.java

```diff
@@ -17,8 +17,6 @@
 import io.prestosql.operator.WindowInfo.DriverWindowInfo;
```

Comment:
can you add test that runs:
```
SELECT * FROM (
        SELECT custkey, max(orderdate) OVER (partition by custkey) FROM tpch.tiny.orders
        UNION ALL (
                SELECT custkey, max(orderdate) OVER (partition by custkey) FROM tpch.tiny.orders
                UNION ALL
                SELECT custkey, max(orderdate) OVER (partition by custkey) FROM tpch.tiny.orders
                UNION ALL ...
        )
) LIMIT 1
```

Test could be disabled if takes more than a minute. Also it could only enabled for single connector so it should not affect tests that much.

How long does it take to execute?

Reply:
I have written such test, but it fails only 1 out of 10 times. When I'm trying to make it more reliable, it takes minutes to finish and there is no guaranty that it won't be flaky even then. I've put the query example in the commit message, so that someone will be able to reproduce this issue, but I am reluctant to add flaky test that will most probably not detect a regression as it get reintroduced.